README.rst

```diff
@@ -1,3 +1,7 @@
+.. image:: logo.svg
```

Comment:
make sure it's being bundled in the package, since it's part of the description, and we'll have to make sure it'll render properly. 
References: https://pypi.org/project/pytest/

https://raw.githubusercontent.com/pytest-dev/pytest/master/README.rst